_posts/en/newsletters/2025-12-05-newsletter.md

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Bitcoin Optech Newsletter #383'
+permalink: /en/newsletters/2025/12/05/
+name: 2025-12-05-newsletter
+slug: 2025-12-05-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+
+FIXME:bitschmidty
+
+## News
+
+- **Consensus bug on NBitcoin: out-of-bound issue in `Remove()`:** Bruno
+  Garcia [posted][bruno delving] to Delving Bitcoin about a theoretical
+  consensus failure in NBitcoin that could occur when using `OP_NIP`. In
+  short when the underlying array is at full capacity and
+  `_stack.Remove(-2)`is called, the Remove operation deletes the item at
+  index 14 and then attempts to shift the subsequent elements down. During
+  this shift the implementation may try and access `_array[16]`, which does
+  not exist leading to an exception. This bug was only found because of
+  [differential fuzzing][diff fuzz] and since the failure is caught in a
```

Comment:
```suggestion
  [differential fuzzing][diff fuzz], and since the failure is caught in a
```

Reply:
thanks, fixed in [220eaa0](https://github.com/bitcoinops/bitcoinops.github.io/pull/2569/commits/220eaa010529751992f355bdc69438ad1a06025d)